src/interface/interface.ts

```diff
@@ -67,6 +78,8 @@ export interface ProvingParams {
   md5: string;
   public_inputs: Array<string>;
   private_inputs: Array<string>;
+  input_context: string; // Hex string of the input context bytes
```

Comment:
why we have both initial_context and input_context? There should be only one parameter to store the context when setup image?

Any reason AddImageParams  has input_context_type?

For setup image, it does not have logic about how to initial context? If user inputs are empty, just use the hardcode default vec!(0), if not, use the user input?